ecommerce/models.py

```diff
@@ -231,6 +231,9 @@ class Coupon(Model):
     )
     enabled = BooleanField(default=True, help_text="If true, coupons are presently redeemable")
 
+    created_on = DateTimeField(auto_now_add=True)
```

Comment:
I think we have a standard `TimestampedModel` in `micromasters.models` that has these, is there a reason not to use that?